kotlin/codegen/pom.xml

```diff
@@ -180,6 +190,11 @@
                   <pattern>com.squareup.kotlinpoet</pattern>
                   <shadedPattern>com.squareup.moshi.kotlinpoet</shadedPattern>
                 </relocation>
+                <!-- Repackage Kotlinx-metadata until its API is stable. -->
```

Comment:
We do need to do something to wire resources correctly still since it uses an SPI. Maybe that's just as simple as manually writing one that points to the new one and excluding the original one we get in the dependency